### ISSUE TITLE
Custom notifications based on output

### ISSUE THREAD

**Issue by OJ7** [jaredly/vscode-background-terminal-notifier#3]
I'm migrating to VS Code's terminal and glad to have found this extension. I was previously using [this method](https://blog.liplex.de/native-osx-notifications-from-iterm-with-triggers/) in iTerm2 which allows having a custom notification show based on certain output in the terminal. 

For example, when running unit tests, I have the results in the notification using regex:
Regex: `(TOTAL: (((\\d+)\\b FAILED\\b.{0,2})|((\\d+)\\b SUCCESS\\b.{0,2}))+)`
Notification: `Unit Tests Results \\1`

This feature request is to allow the user to specify an output string to listen to and have an associated custom text notification display.

**Comment by jaredly** [comment]
Yeah, having access to the command's output would be really cool! Unfortunately VSCode doesn't provide access to the contents of the console :( it only gives me the PID of the shell that's running, and then I use PS to determine what child processes are running.
